src/main/java/org/haffson/adventofcode/days/day06/DaysConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package org.haffson.adventofcode.days.day06;
+
+import org.haffson.adventofcode.utils.FileReaders;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.Nonnull;
+
+@Configuration
+public class DaysConfiguration {
+
+    @Bean
+    public Day06 createDay06(@Nonnull final FileReaders fileReaders) {
+        return new Day06(fileReaders, 10000);
+    }
+}
```

Comment:
Du solltest eigentlich immer eine neue Zeile am Ende haben:

```suggestion
}

```

Das gilt für die meisten Files. Bei mir wird das automatisch beim Speichern einer Datei gemacht, was ich für sinnvoll halte (_Settings_ → _Editor_ → _General_ → _Ensure every saved file ends with a line break_). Bei der Gelegenheit kannst du auch _Remove trailing spaces on: All lines_ und _Remnove trailing blank lines at end of saved files_ aktivieren, was beides ebenfalls sinnvolle Optionen sind.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Wie geil ist das denn!! Auf gar keinen Fall habe ich bei meinem letzten Projekt jede Datei einzeln geöffnet und unten eine Zeile hinzugefügt...

Reply:
Willkommen bei _Jans Reviews™_